### PR TITLE
revamp: us/train-new-model

### PR DESCRIPTION
**Pull request by techyNonso**
## Related Issue(s)

Fixes #
Resolves #
Closes #

## Proposed Changes

[//]: # '[List down the specific changes you have made in this pull request.]'

- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [ ] I have added suitable unit tests (if applicable).

## To-Do

- [ ]

### CODE REVIEW

File: compute-samples/boston-housing-price-prediction.zip
Comment (Nasfame) [inline compute-samples/boston-housing-price-prediction.zip:1]: i don't understand this change: @techyNonso compute-samples is already in main

**Comment by Nasfame** [comment]
@techyNonso do merge main branch and request a review

**Comment by techyNonso** [comment]
> @techyNonso do merge main branch and request a review

I don't understand

**Comment by Nasfame** [comment]
@techyNonso verel deployment and build is failing, kindly fix

**Comment by techyNonso** [comment]
> @techyNonso verel deployment and build is failing, kindly fix

IT has been fixed, please review and merge

**Review by Nasfame — APPROVED** [review]
Code LGTM, E2E test failed on compute. But okay, since logs access is gone. Will be raising on a ticket on vercel function failure later

**Pull request merged**